### PR TITLE
fix redirect bug in example

### PR DESCRIPTION
**Pull request by DanielBiegler**
`pathname` already includes a slash which results in a bad redirect

### CODE REVIEW

**Review by leerob — COMMENTED** [review]
https://github.com/vercel/next.js/blob/canary/examples/app-dir-i18n-routing/middleware.ts#L54

**Review by balazsorban44 — APPROVED** [review]

**Review by leerob — CHANGES_REQUESTED** [review]
Please see my comment above :pray:

**Review by leerob — CHANGES_REQUESTED** [review]
Please see my comment above :pray:

File: docs/02-app/01-building-your-application/01-routing/13-internationalization.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export function middleware(request) {
     // e.g. incoming request is /products
     // The new URL is now /en-US/products
     return NextResponse.redirect(
-      new URL(`/${locale}/${pathname}`, request.url)
+      new URL(`/${locale}${pathname}`, request.url)
```
Comment (DanielBiegler) [inline docs/02-app/01-building-your-application/01-routing/13-internationalization.mdx:57]: @leerob you want to change this into the following?:

```suggestion
      new URL(`/${locale}${pathname.startsWith('/') ? '' : '/'}${pathname}`, request.url)
```

**Comment by DanielBiegler** [comment]
image

**Pull request closed without merging**